### PR TITLE
net-misc/frr: fixing multiple vulnerabilities [please reassign]

### PR DESCRIPTION
**Pull request by alarig**
Bug: https://bugs.gentoo.org/913242

### CODE REVIEW

**Comment by jkroonza** [comment]
@alarig you aware of any cause for delay here?

File: net-misc/frr/files/frr-9.0-bug-913242.patch [hunk 1]
```diff
@@ -0,0 +1,173 @@
+diff --git a/bgpd/bgp_attr.c b/bgpd/bgp_attr.c
+index 60e41ac1cc83..7badc6a17cbf 100644
```
Comment (jsmolic) [inline net-misc/frr/files/frr-9.0-bug-913242.patch:2]: Could you please add a link to upstream PR/commit where these are taken from?

**Review by jsmolic — CHANGES_REQUESTED** [review]

File: net-misc/frr/files/frr-9.0-bug-913242.patch (hunk 1 above)
Comment (alarig) [reply net-misc/frr/files/frr-9.0-bug-913242.patch]: I removed 9.0 as 9.0.2 is needed for new CVEs

File: net-misc/frr/frr-9.0.1.ebuild [hunk 2]
```diff
@@ -14,7 +14,7 @@ S="${WORKDIR}/frr-${P}"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~arm64 ~x86"
+KEYWORDS="amd64 ~arm64 ~x86"
```
Comment (juippis) [inline net-misc/frr/frr-9.0.1.ebuild:17]: We don't let users stabilize packages, but it should go through arch testers. Can you file a stabilization bug in bugzilla? Let me know if you're not familiar with that.

Oh and if I read the CVE correctly, 9.0.1 is still affected. So we should insta-stabilize 9.0.2 instead?

**Review by juippis — APPROVED** [review]

**Comment by juippis** [comment]
I'm gonna merge the version bumps - let's stabilize 9.0.2 immediately.

File: net-misc/frr/frr-9.0.1.ebuild (hunk 2 above)
Comment (alarig) [reply net-misc/frr/frr-9.0.1.ebuild:17]: We should, I just didn’t want to stabilise 9.0.2 right away :)

**Pull request closed without merging**